packages/storybook/src/stories/ino-input-file/ino-input-file.stories.ts

```diff
@@ -32,37 +34,47 @@ export default {
       handles: ['changeFile .customizable-input'],
     },
   },
-} as Meta;
+  args: {
+    accept: '',
+    autoFocus: false,
+    disabled: false,
+    multiple: false,
+    required: false,
+    label: 'Select a file to upload',
+    labelSelected: 'selected',
+    dragAndDrop: false,
+    dragAndDropText: 'Drag your files here',
+    dragAndDropSecondaryText: 'or',
+  }
+} as Meta<Components.InoInputFile>;
 
-export const Playground = (args): TemplateResult => html`
+
+type InoInputFileExtended = Components.InoInputFile & {
+  labelSelected: string,
+}
+
+const template = new TemplateGenerator<InoInputFileExtended>(
+  'ino-input-file',
+  args => html`
   <ino-input-file
-    class="customizable-input"
-    accept="${args.accept}"
-    autofocus="${args.autofocus}"
-    disabled="${args.disabled}"
-    multiple="${args.multiple}"
-    required="${args.required}"
-    label="${args.label}"
-    label-selected="${args.labelSelected}"
-    drag-and-drop="${args.dragAndDrop}"
-    drag-and-drop-text="${args.dragAndDropText}"
-    drag-and-drop-secondary-text="${args.dragAndDropSecondaryText}"
-  >
-  </ino-input-file>
-`;
-Playground.args = {
-  accept: '',
-  autoFocus: false,
-  disabled: false,
-  multiple: false,
-  required: false,
-  label: 'Select a file to upload',
-  labelSelected: 'selected',
-  dragAndDrop: false,
-  dragAndDropText: 'Drag your files here',
-  dragAndDropSecondaryText: 'or',
-};
+  class="customizable-input"
+  accept="${args.accept}"
+  autoFocus="${args.autoFocus}"
+  disabled="${args.disabled}"
+  multiple="${args.multiple}"
+  required="${args.required}"
+  label="${args.label}"
+  label-selected="${args.labelSelected}"
+  drag-and-drop="${args.dragAndDrop}"
+  drag-and-drop-text="${args.dragAndDropText}"
+  drag-and-drop-secondary-text="${args.dragAndDropSecondaryText}"></ino-input-file>
+`);
+export const Playground = template.generatePlaygroundStory();
+
 
-export const DragAndDrop = (): TemplateResult => html`
+const templateDragAndDrop = new TemplateGenerator<InoInputFileExtended>(
+  'ino-input-file',
+  args => html`
   <ino-input-file multiple drag-and-drop></ino-input-file>
-`;
+`);
+export const DragAndDrop = templateDragAndDrop.generatePlaygroundStory();
```

Comment:
Instead of writing another template, we could just reuse the playground one:
```suggestion
export const DragAndDrop = template.generateStoryForProp('dragAndDrop', true);
```